src/Operators/finitedifference.jl

```diff
@@ -1588,6 +1588,210 @@ end
     return Geometry.Contravariant3Vector(vᶠ)
 end
 
+
+
+#########################
+"""
+    U = FCTZalesak(;boundaries)
+    U.(A, Φ, Φᵗᵈ)
+
+Correct the flux using the flux-corrected transport formulation by Zalesak
+Input arguments:
+- a face-valued vector field `A`
+- a center-valued field `Φ`
+- a center-valued field `Φᵗᵈ`
+```math
+Φ_j^{n+1} = Φ_j^{td} - \frac{dt}{dx}(C_{j+\frac{1}{2}}A_{j+\frac{1}{2}} - C_{j-\frac{1}{2}}A_{j-\frac{1}{2}})
+```
+This stencil is based on [zalesak1979fully](@cite).
+
+Supported boundary conditions are:
+- [`ThirdOrderOneSided(x₀)`](@ref): uses the third-order downwind reconstruction to compute `x` on the left boundary,
+and the third-order upwind reconstruction to compute `x` on the right boundary.
```

Comment:
If in this case, the boundary condition is falling back to the first-order-upwinding case, then this docstring should be updated

Reply:
thanks! just change it to FirstOrder